inst/WORDLIST

```diff
@@ -74,15 +76,23 @@ minimise
 msg
 nano
 organisation
+<<<<<<< HEAD
 organisations
 pkgbuild
+=======
```

Comment:
looks like you have some unresolved conflict markers here

Reply:
Strange. I didn't think Git would allow me to add the file and `git rebase --continue` if the file still contained the conflict markers. I'll fix that real quick.

Reply:
No by default, but you can use `git diff --check` to warn about them, or put this in a pre-commit hook if you want to automate it.